### DISCUSSION TITLE
Change position marker behaviour

### DISCUSSION THREAD

**Discussion by SergSel2006** [organicmaps/organicmaps#3530] [Ideas]
I was just looking at my maps and was constantly distracted by my position marker. As I hold my phone vertically while looking at it and stop moving, application will use hardware compass, which in vertical state will rotate as  randomly as marker rotates. It is small problem, but I'd like to get just a circle instead of arrow while standing and looking at map vertically, and if phone in horizontal position, only then use arrow as position marker.

**Comment by biodranik** [comment]
So the issue is that the blue arrow moves randomly when you don't move and the device is not moved, right?

1. Are there any metal objects or radio/magnetic/electric devices around that may interfere with the compass?
2. Did you calibrate your compass on the device? Did you rotate the device in a special "8"-like movement? There are some articles on the Internet about it.

**Reply by SergSel2006** [reply]
Hello again. you understand me right. I've seen some problems with compass sensor on my device, after calibrating it still can move, but not that much. So, it is problem on my side really. Thanks for your time anyways.

**Reply by biodranik** [reply]
Please let us know if you find a way to "tame" it (almost) completely :)
